crates/op-rbuilder/src/builders/flashblocks/best_txs.rs

```diff
@@ -21,6 +21,13 @@ where
     inner: I,
     current_flashblock_number: Arc<AtomicU64>,
     early_transactions: BTreeMap<u64, VecDeque<T>>,
+    // Transactions that were already commited to the state. Using them again would cause NonceTooLow
+    // so we skip them
+    commited_transactions: HashSet<TxHash>,
+    // List of transaction that were marked invalid on previous iterations.
+    // We keep it to not reexecute them multiple times
+    // TODO: Maybe it make sense to reexecute them
+    invalid_transaction: Vec<(Address, u64)>,
```

Comment:
when do they get marked as invalid? I believe it is when the nonce is too high / or evm execution error?

I think we can reexecute the nonce tx in case the lower nonce tx comes in midflashblock but not sure if invalid can differentiate the error type

Reply:
During best_transaction_execution
There is various reasons for this, for example insufficient gas.
I will remove this, because there is mix between deterministic reason and non-determinitic